chore: remove Express restaurant because it's gone in reality

Jassob opened the pull request:
It's good if mat reflects reality. This commit makes mat more reality reflective and fixes #110.

Thank you Rembane for the above commit message.

Jassob committed:
chore: remove Express restaurant because it's gone in reality

It's good if mat reflects reality. This commit makes mat more reality
reflective and closes #110.

Thank you @Rembane for the above commit message.

The1Penguin commented on test/Main.hs:
```diff
@@ -24,24 +24,6 @@ testFun expected = either
-          "{\"data\":{\"dishOccurrencesByTimeRange\":[{\"displayNames\":[{\"name\":\"Kycklinggryta, persilja, senap & kokt potatis\",\"categoryName\":\"Swedish\"},{\"name\":\"Chicken stew, parsley, mustard & boiled potatoes\",\"categoryName\":\"English\"}],\"startDate\":\"9/9/2019 12:00:00 AM\",\"dishType\":{\"name\":\"Express\"},\"dish\":{\"name\":\"Kycklinggryta, aubergin, zucchini, tomat & ros ...
-        )
-    )
-
```
While I agree with the premise of this PR, test for Kåren would still be good, one could convert it for current Kårresturangen

Jassob replied:
Right, then I'll fix that.

I don't remember enough of this repo to know if Express was part of Kåren or not, but your absolutely right!

Rembane replied:
I got there first! :D Is fix! 

https://github.com/dtekcth/mat-chalmers/pull/111/commits/41d04a1efd647c1b56a0fff65ba3ec325268f83e

Rembane committed:
chore: Put back Kåren-test with fresh data

Thank you @The1Penguin for the feedback!

Run this in the repl to get fresh test data:

```Haskell
dateNow <- fmap (view _zonedTimeToLocalTime) getZonedTime
tomorrow = (dateNow & (_localDay . gregorian . _ymdDay) %~ (+ 1)) ^. _localDay
runStack $ fetch "21f31565-5c2b-4b47-d2a1-08d558129279" tomorrow
```

Then run the test and adjust the expected data to match the actual data.
This is the backwards way to write tests, but I think it's alright when
it's this easy to see if expectation and actual data matches.

The1Penguin reviewed (approved):
LGTM! :+1:
